Bottom: 98755dd20c0103988a0feea778739a2cf4ab626a
Top:    5debe9fe8e9952bd520df07b40c18851ff0981c6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 19:31:21 +0100

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/draw.c b/src/draw.c
index 994dd1c..6c3d709 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1168,10 +1168,6 @@ DrawLayerGroup (int group, const BoxType *drawn_area)
         rv = 0;
       if (layernum < max_copper_layer && Layer->On)
         DrawLayerCommon (Layer, drawn_area, true);
-
-      /* HACK: Let GTK/GL subcomposite each layer in the group separately */
-      if (gui->gui)
-        gui->set_layer (0, group, 0);
     }
   if (n_entries > 1)
     rv = 1;
